addons/website_hr_recruitment/controllers/main.py

```diff
@@ -74,8 +74,8 @@ def jobs(self, country=None, department=None, office_id=None, **kwargs):
 
     @http.route('/jobs/add', type='http', auth="user", website=True)
     def jobs_add(self, **kwargs):
-        # avoid branding of website_description by setting rendering_bundle in context
-        job = request.env['hr.job'].with_context(rendering_bundle=True).create({
+        # avoid branding of website_description by setting inherit_branding in context
+        job = request.env['hr.job'].with_context(inherit_branding=False).create({
```

Comment:
Not sure it is correct to replace `rendering_bundle=True` by `inherit_branding=False`